modules/utxo-lib/src/transaction_builder.ts

```diff
@@ -1,5 +1,5 @@
 import * as types from 'bitcoinjs-lib/src/types';
-import { address as baddress } from './';
+import { address as baddress, p2tr, p2tr_ns } from './';
```

Comment:
https://github.com/BitGo/BitGoJS/pull/3278/files#diff-fa1283e60c39bb1ebf8337d3389acc9246b1e313c3d7bb50c6c8967ee4e3f45dR15

Had to export payments module as `p2trPayments` since we're doing `export * from bitcoinjs-lib` which is exporting a `payments` module.